integration/misc/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "mud-integration-tests-misc",
```

Comment:
do we expect more than just one here? will it be useful to have isolated integration projects?

if so, it would be nice to come up with a better name than "misc"

if not, we could just make `integration` the project root (or better yet: `integration-tests`)

Reply:
although I could just have a 2nd contracts package, it doesn't need to be multiple projects, will move it to root for now

Reply:
thoughts on renaming the integration dir to integration-sandbox to match?